src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -70,6 +72,19 @@ public StorageFile(String filePath) throws InvalidStorageFilePathException {
         }
     }
 
+    public boolean exist() {
+    	File f = new File(DEFAULT_STORAGE_FILEPATH);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.